app/javascript/dashboard/routes/dashboard/settings/inbox/Settings.vue

```diff
@@ -115,13 +130,13 @@ export default {
       }
 
       if (
-        this.isATwilioChannel ||
-        this.isALineChannel ||
+        isATwilioChannel(this.inbox) ||
+        isALineChannel(this.inbox) ||
         this.isAPIInbox ||
         (this.isAnEmailChannel && !this.inbox.provider) ||
```

Comment:
@fayazara Does this `this.isAPIInbox` and `this.isAnEmailChannel` works? We need to pass the inbox to the helper, right?